dfastmi/io/RiverParameterData.py

```diff
@@ -29,18 +29,20 @@
 
 import configparser
 from typing import Tuple, List, Optional
+from dfastmi.io.Reach import Reach
+from dfastmi.io.Branch import Branch
 
 class RiverParameterData:
     """
         Collect river parameter data from river configuration object.
     """
 
-    _branches: List[str]
+    #_branches: List[str]
     """
        The list of river branches. The length of this list is nBranches.
     """
 
-    _nreaches: List[int]
+    #_nreaches: List[int]
```

Comment:
whats the plan with this commented part?

Reply:
will be removed

Reply:
Discussed, we will leave these comments for a later refactoring.
This will be tackled in https://issuetracker.deltares.nl/browse/DFAST-112